dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -554,6 +562,16 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
         return services;
     }
 
+    private List<Workflow> getBioworkflows(User user) {
+        return serviceDAO.findMyEntries(user.getId()).stream().map(BioWorkflow.class::cast).collect(Collectors.toList());
```

Comment:
Does this work, it looks like you're asking the serviceDAO for workflows when it seems like workflowDAO is what I would have expected. Tests are passing though ... which might mean that this needs tests?

Reply:
Good catch, I meant to do workflowDAO, was a copy paste error. The return value of the API wasn't directly tested, as I had another API call to grab it. Not sure why as this function would return the same thing anyways. I'll fix this so that it is properly tested.